### PR TITLE
Update browser support

### LINKED ISSUE #830: When there be full support for Edge browser
As noted in the readme there is no mention of Edge support?  It would be great to support.

### PR DESCRIPTION
**Pull request by dylanb**
fixes #830

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,12 +63,17 @@ axe.run(function (err, results) {
...
 
-The [aXe API](doc/API.md) supports the following browsers:
+The [aXe API](doc/API.md) fully supports the following browsers:
 
-* Internet Explorer v9, 10, 11
+* Microsoft Edge v40 and above
 * Google Chrome v42 and above
 * Mozilla Firefox v38 and above
 * Apple Safari v7 and above
+* Internet Explorer v9, 10, 11
```
Comment (WilcoFiers) [inline README.md]: I'd be massively in favour of dropping support for all but IE 11.

**Comment by nschonni** [comment]
yes, iE 10 is only supported officially on Server 2008 R1 no, and support for that is goes away soonish                                                                                                                                                                        Mistyped on my CrackBerry                                                                                                                                                                           From: notifications@github.comSent: April 6, 2018 4:02 PMTo: axe-core@noreply.github.comReply to: reply@reply.github.comCc: subscribed@noreply.github.comSubject: Re: [dequelabs/axe-core] Update browser support (#836)  @WilcoFiers commented on this pull request.



In README.md:
>  * Google Chrome v42 and above
 * Mozilla Firefox v38 and above
 * Apple Safari v7 and above
+* Internet Explorer v9, 10, 11

I'd be massively in favour of dropping support for all but IE 11.

File: README.md (hunk 1 above)
Comment (dylanb) [reply README.md]: It is still very widely used in government

Comment (WilcoFiers) [reply README.md]: "very widely" seems a bit much. We have analytics on DQU right? How much IE9 + IE10 users do we get on the axe-core pages there?

Comment (WilcoFiers) [reply README.md]: I looked up our analytics. We've got 0.3% traffic on IE9, we have nothing on IE10. As far as older IE support, there is still 1.9% on IE8, and 0.4% on IE7, but we've never supported those. I think it's pretty reasonable to drop IE9 & 10 at this point.

Comment (dylanb) [reply README.md]: The discussion/decision about whether to drop support for IE<11 should not hold up this PR

**Pull request merged**